frontend/src/app/filter/filter-manager.component.ts

```diff
@@ -15,6 +16,11 @@ import { FilterInterface, QueryModel } from '../models/index';
 export class FilterManagerComponent {
     @Input() queryModel: QueryModel;
 
+    icons = {
+        clear: faTrashCan,
+        toggle: faFilter,
+    };
+
```

Comment:
can we factor this out? This is now repeated here and in `filter-box-component`.